### PR TITLE
adding a new game dodge teh blocks

### LINKED ISSUE #4751: [New game]: adding new game Dodge the Blocks
### 🎮 Game Request

Game logic and basic description: In this game, the player controls a character that must avoid falling blocks. The goal is to survive as long as possible without being hit

### Point down the features

Game points

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by 1911aditi**
## PR Description 📜

In this game, the player controls a character that must avoid falling blocks. The goal is to survive as long as possible without being hit.

Fixes #4751

<hr>
 
##4751 ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

Screenshot 2024-07-06 200816

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @1911aditi ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

File: assets/images/Screenshot 2024-07-06 200816.png
Comment (ishita-43) [inline assets/images/Screenshot 2024-07-06 200816.png:1]: Hi please rename the image as per contribution guidelines for naming conventions

**Review by ishita-43 — CHANGES_REQUESTED** [review]
I HOpe you will make the desired changes soon

File: assets/images/Screenshot 2024-07-06 200816.png
Comment (ishita-43) [reply assets/images/Screenshot 2024-07-06 200816.png:1]: you need to change the name of this image also as dodge_the_blocks.png

**Comment by 1911aditi** [comment]
can u check now?

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @1911aditi , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**